wbooks/src/app/index.tsx

```diff
@@ -5,15 +5,25 @@ import BookDetail from '@screens/BookDetail';
 import { NavigationContainer } from '@react-navigation/native';
 import { createStackNavigator } from '@react-navigation/stack';
 import routes from '@constants/routes';
+import { white } from '@constants/colors';
+import HeaderBackButton from '@components/HeaderBackButton';
+
+import styles from './styles';
 
 function App() {
   const Stack = createStackNavigator();
 
+  const headerStyle = {
+    headerStyle: styles.headerStyle,
+    headerTintColor: white,
+    headerLeft: () => <HeaderBackButton />
+  };
```

Comment:
Mova to a constants file. Also, and the `headerBackgroud`? You must use an ImageBackground, not a color directly 🤔